src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -1916,7 +1916,10 @@ public virtual bool Accepts(DType type, out KeyValuePair<string, DType> schemaDi
                 case DKind.String:
                     accepts =
                         type.Kind == Kind ||
-                        type.Kind == DKind.Hyperlink ||
+
+                        // !!!???
+                        //type.Kind == DKind.Hyperlink || 
```

Comment:
This will break PA because they don't go through IR. 
Can we have IR inject a coercion regardless of Accepts?